src/language-js/print/array.js

```diff
@@ -170,6 +171,18 @@ function isConciselyPrintedArray(node, options) {
   );
 }
 
+function isLineAfterElementEmpty(path, { originalText: text }) {
```

Comment:
Can we use destructuring assignment here?

```suggestion
function isLineAfterElementEmpty({ node }, { originalText: text }) {
```